packages/ui/src/components/page/Sidebar/CurrentMember/CurrentMember.tsx

```diff
@@ -10,19 +10,24 @@ import { AddMembershipButton } from '../../../membership/AddMembershipButton'
 import { SwitchMemberModal } from './SwitchMemberModal'
 
 export const CurrentMember = () => {
-  const { count, members, active } = useMyMemberships()
+  const { count, members, active, setActive } = useMyMemberships()
   const [isOpen, toggleOpen] = useToggle()
   const [isCreateOpen, toggleCreateOpen] = useToggle()
+  useEffect(() => {
+    count == 1 && setActive(members[0])
+  }, [count])
 
   if (count < 1) {
     return <AddMembershipButton>Create a membership</AddMembershipButton>
   }
 
+  count > 1 && !active && !isOpen && toggleOpen()
+
   return (
     <>
       <MembershipsCount />
       <SwitchMember onClick={toggleOpen}>
-        <MemberInfo member={active || members[0]} isOnDark={true} />
+        {active && <MemberInfo member={active} isOnDark={true} />}
```

Comment:
To shed more light on my previous comment.

This notation is OK as this is both

*   single `&&`
*   and outside "logic" part of the component (layout part)